packages/terra-hookshot/src/Hookshot.jsx

```diff
@@ -0,0 +1,336 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from 'react-portal';
+import HookshotContent from './_HookshotContent';
+import HookshotUtils from './_HookshotUtils';
+
+const ATTACHMENT_POSITIONS = [
+  'top left',
+  'top center',
+  'top right',
+  'middle left',
+  'middle center',
+  'middle right',
+  'bottom left',
+  'bottom center',
+  'bottom right',
+];
+
+const ATTACHMENT_BEHAVIORS = [
+  'auto',
+  'flip',
+  'none',
+];
+
+const propTypes = {
+  /**
+   * If the primary attachment in not available, how should the content be positioned.
+   * Valid values: 'auto', 'flip', and 'none'.
+   */
+  attachmentBehavior: PropTypes.oneOf(ATTACHMENT_BEHAVIORS),
+  /**
+   * Value in px of the margin to place between the target and the content.
+   */
+  attachmentMargin: PropTypes.number,
+  /**
+   * Reference to the bounding container, wil use window if nothing is provided.
+   */
+  boundingRef: PropTypes.func,
+  /**
+   * The HookshotContent to be attached.
+   */
+  content: PropTypes.element.isRequired,
+  /**
+   * String pair seperated by a space using values of top, middle, bottom, and left, center, right.
```

Comment:
*separated.

Also, I think providing an example in the documentation comment can help make this easier to understand. (ex: 'top middle'